CONTRIBUTING.md

```diff
@@ -1,6 +1,3 @@
-# To Do // To improve ?
```

Comment:
@midudev detectaba el # del "to do improve" como parte del markdown y se reflejaba con todo lo demas al ir a leerlo de manera normal..
Por eso lo elimine 😅

<img width="1033" alt="Captura de pantalla 2023-01-06 a las 10 43 00" src="https://user-images.githubusercontent.com/77696223/210974700-cb9d56d6-a267-431e-a98d-66dbea040d14.png">